produc_demo.rb

```diff
@@ -0,0 +1,12 @@
+require_relative("product.rb")
```

Comment:
There is no need for brackets and try to use double quotes " only when it is needed. In all other cases please use ' .
In this case it can be 
```ruby
require_relative 'product'
```